external/loaders/loaders/_config.py

```diff
@@ -131,3 +145,10 @@ def load_batches(self, variables) -> Batches:
         """
         batches_function = batches_functions[self.batches_function]
         return batches_function(self.data_path, list(variables), **self.batches_kwargs,)
+
+    def __post_init__(self):
```

Comment:
Should this already have inherited its parent class `BatchesLoader`'s `__post_init__`?

Reply:
BatchesLoader does not have a `__post_init__` defined.